package.json

```diff
@@ -30,6 +30,9 @@
       }
     }
   },
+  "overrides": {
+    "@digitalbazaar/http-client": "4.1.0"
```

Comment:
there were 2 versions of this installed, because `@interop/did-web-resolver` also depends on a different version of http-client